### ISSUE TITLE
Failures on Expose CommentToken as analyzer public API...[analyzer]

### ISSUE THREAD

**Issue by a-siva** [dart-lang/sdk#56061]
There are new test failures on [Expose CommentToken as analyzer public API...[analyzer] (https://github.com/dart-lang/sdk/compare/c001d5116abade6a1861b1e7903b9efd08d4582e~...88e62a23cfb1f757a9648b23457132754815750c).

The tests
```
pkg/dartdev/test/commands/analyze_test RuntimeError (expected Pass)
```
are failing on configurations
```
unittest-asserts-release-win-arm64
```

```
00:38 [32m+35[0m[33m ~1[0m[31m -1[0m: analyze display mode default does not prioritize warnings in analysis_options.yaml [1m[31m[E][0m[0m

  Expected: 'Analyzing myapp...\n'
              '\n'
              '  error - lib\\main.dart:1:16 - A value of type \'Null\' can\'t be returned from the function \'foo\' because it has a return type of \'int\'. - return_of_invalid_type\n'
              'warning - analysis_options.yaml:1:10 - The include file \'package:lints/recommended.yaml\' in \'c:\\opt\\s\\w\\ir\\x\\t\\dartdevf46a7db6\\myapp\\analysis_options.yaml\' can\'t be found when analyzing \'c:\\opt\\s\\w\\ir\\x\\t\\dartdevf46a7db6\\myapp\'. - include_file_not_found\n'
              '\n'
              '2 issues found.'
    Actual: 'Analyzing myapp...\n'
              '\n'
              '  error - lib\\main.dart:1:16 - A value of type \'Null\' can\'t be returned from the function \'foo\' because it has a return type of \'int\'. - return_of_invalid_type\n'
              'warning - analysis_options.yaml:1:10 - The include file \'package:lints/recommended.yaml\' in \'C:\\opt\\s\\w\\ir\\x\\t\\dartdevf46a7db6\\myapp\\analysis_options.yaml\' can\'t be found when analyzing \'C:\\opt\\s\\w\\ir\\x\\t\\dartdevf46a7db6\\myapp\'. - include_file_not_found\n'
              '\n'
              '2 issues found.'
     Which: is different.
            Expected: ... yaml' in 'c:\\opt\\s ...
              Actual: ... yaml' in 'C:\\opt\\s ...
                                    ^
             Differ at offset 277
  

  package:matcher/src/expect/expect.dart 149:31      fail
  package:matcher/src/expect/expect.dart 144:3       _expect
  package:matcher/src/expect/expect.dart 56:3        expect
  pkg\dartdev\test\commands\analyze_test.dart 664:9  defineAnalyze.<fn>.<fn>.<fn>
  ===== asynchronous gap ===========================
  package:test_api/src/backend/declarer.dart 215:9   Declarer.test.<fn>.<fn>
  ===== asynchronous gap ===========================
  package:test_api/src/backend/declarer.dart 213:7   Declarer.test.<fn>
  ===== asynchronous gap ===========================
  package:test_api/src/backend/invoker.dart 258:9    Invoker._waitForOutstandingCallbacks.<fn>
```

**Comment by a-siva** [comment]
// @srawlins

**Comment by bwilkerson** [comment]
Closing as a duplicate of https://github.com/dart-lang/sdk/issues/56059.

**Comment by bwilkerson** [comment]
Should be fixed by [c34f793](https://github.com/dart-lang/sdk/commit/c34f79355fd859992a5626baa80f49593f06e139).
